packages/usa-character-count/src/index.js

```diff
@@ -28,13 +35,25 @@ const getCharacterCountElements = (inputEl) => {
     throw new Error(`${INPUT} is missing outer ${CHARACTER_COUNT}`);
   }
 
+  const formGroupEl = characterCountEl.querySelector(FORM_GROUP);
+
+  if (!formGroupEl) {
+    throw new Error(`${CHARACTER_COUNT} is missing inner ${FORM_GROUP}`);
+  }
+
+  const labelEl = characterCountEl.querySelector(LABEL);
+
+  if (!labelEl) {
+    throw new Error(`${CHARACTER_COUNT} is missing inner ${LABEL}`);
+  }
+
```

Comment:
> [!NOTE]
> Added checks for form group and label elements. These elements are necessary for the overall structure of the component and will result in JS errors for `toggleErrorState` if they are missing.

Reply:
Removed errors in [fe42118](https://github.com/uswds/uswds/pull/5908/commits/fe4211882a16f30e80c4559bb970e9b3119dfdf4)